src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleGroupSyncMonitor.scala

```diff
@@ -130,7 +131,7 @@ class GoogleGroupSyncMonitorActor(
       val groupId: WorkbenchGroupIdentity = parseMessage(message)
 
       groupSynchronizer
-        .synchronizeGroupMembers(groupId)
+        .synchronizeGroupMembers(groupId, samRequestContext = SamRequestContext(null))
```

Comment:
does the `GoogleGroupSyncMonitorActor` need to have tracing?

Reply:
I think it would be good to have tracing here but could be another ticket and definitely a follow on PR.

Reply:
New ticket: https://broadworkbench.atlassian.net/browse/CA-817

Should I start working on this immediately or should it be prioritized in the backlog?

Reply:
I think it's like 3 lines of code, but we can separate it.
https://opencensus.io/quickstart/java/tracing/#using-the-tracer